cylc/flow/suite_srv_files_mgr.py

```diff
@@ -63,6 +63,8 @@ class SuiteSrvFilesManager(object):
     PASSPHRASE_LEN = 20
     PS_OPTS = '-opid,args'
     REC_TITLE = re.compile(r"^\s*title\s*=\s*(.*)\s*$")
+    REG_PATTERN = r'^[\w_+@](?:[\w_+@\.-]+)?$'
```

Comment:
Need the `/`? Note comment by @sadielbartholomew on matching the character set of task name? (From `cylc.flow.task_id`?)

Reply:
To permit hierarchical suite names, how about:

```suggestion
    REG_PATTERN = r'^[\w_+@](?:[\w\/_+@\.-]+)?$'
```

Don't see how this relates to the `cylc.flow.task_id` comment though.